### ISSUE TITLE
Connect to mysql via tunnel script

### ISSUE THREAD

**Issue by shanke** [TablePlus/TablePlus#1246]
Navicat Premium has the ability to connect over http via a tunnel-script.

Is it possible to implement such a feature?

**Comment by RomkaLTU** [comment]
Yes, very handy feature, when you only have FTP access to shared hosting or something.

**Comment by garrettmassey1** [comment]
+1, I have tried to figure out a workaround for this, but can't get anything to work with our production servers unless we have the ability to use http tunneling.
